finquant/portfolio.py

```diff
@@ -85,9 +86,11 @@ def __init__(self):
         self.expected_return = None
         self.volatility = None
         self.sharpe = None
+        self.var = None
         self.skew = None
         self.kurtosis = None
         self.totalinvestment = None
+        self.var_confidence_level = 0.95
```

Comment:
Just a question about this (as I don't know about it): Is 0.95 a common value for the var confidence level?

Reply:
Apparently, the most common values in quantitative finance are 0.95 and 0.99. The course I am following uses 0.95 by default in all the examples

Reply:
That works for me then, thank you :)